docs/data/date-pickers/fields/fields.md

```diff
@@ -17,7 +17,7 @@ They might receive breaking changes on their props to have the best components p
 
 ## Introduction
 
-The fields are React components that let you enter a date or time with the keyboard, without using any modal or dropdown UI.
```

Comment:
I went for `popover` everywhere instead of `dropdown` because it seem to be the term used by the core (https://mui.com/material-ui/react-popover/#main-content)